src/dosbox.cpp

```diff
@@ -846,6 +845,9 @@ void DOSBOX_Init()
 	// Configure Gravis UltraSound emulation
 	GUS_AddConfigSection(control);
 
+	// Configure the IBM Music Feature emulation
```

Comment:
"IBM Music Feature Card"
In general we should not allow card in the name (so no "Sound Blaster card", "Gravis Ultrasound card"), but in this case "card" is part of the official name and abbreviation.

Reply:
![2023-03-27_21-55](https://user-images.githubusercontent.com/1557255/228132269-bad26b56-f69d-444f-9746-7a33553c325c.png)

Reply:
https://ibm-pc.org/manuals/ibm/options/IMFC_Manual.pdf

IBM's title for it is "IBM PC Music Feature", but lots of installers tacked on the "card".  :man_shrugging: 

The files were originally named 'imfc.cpp' - so it's not a big deal going back to IMFC (there's a commit where I actually switch over to dropping the card).

Reply:
> IBM's title for it is "IBM PC Music Feature", but lots of installers tacked on the "card". 🤷‍♂️

Are we here emulating both the card and the MIDI Adapter Unit?

Reply:
If the code only tackles emulating the music card portion, I would still recommend @johnnovak 's suggestion.

Reply:
Sierra driver also calls it IMF:

![2023-03-27_22-24](https://user-images.githubusercontent.com/1557255/228136561-744f2bbf-7ab8-49ee-a7d9-d32b8a24faf0.png)

Reply:
After hearing LGR's explanation, I think I agree even more with @johnnovak here. It should be "IBM Music Feature Card" instead of "IBM Music Feature", because I believe the "IBM Music Feature " encompasses the MIDI Adapter Unit as well!

Reply:
That's a great point, @ThomasEricB - and that makes a lot of sense.

And you're right: this is emulating the IMF Card, and not the external MIDI breakout box (which would require hooking up an FB-01 device or using an FB-01 emulator).

Will move it back to IMFC.